tests/integration/hive_metastore/test_workflows.py

```diff
@@ -11,11 +12,11 @@
         ("regular", "migrate-tables"),
         ("hiveserde", "migrate-external-hiveserde-tables-in-place-experimental"),
         ("hiveserde", "migrate-external-tables-ctas"),
+        # TODO: Some workflows are missing here, and also need to be included in the tests.
```

Comment:
Add them

Reply:
The omitted workflows can't be trivially added to this list, and this was something I noted while working on the PR rather than a problem being introduced.

I've created #3422 to cover the integration tests for workflows that aren't being tested.